presto-main/src/main/resources/webapp/dist/index.js

```diff
@@ -106,7 +106,7 @@ eval("\n\nObject.defineProperty(exports, \"__esModule\", {\n    value: true\n});
                                       { className: \"uppercase\" },\n                                            \"Uptime\"\n                                        ),\n                                        _react2.default.createElement(\"br\", null),\n                                        _react2.default.createElement(\n                                            \"span\",\n                                            { \"data-toggle\": \"tooltip\", \"data-placement\": \"bottom\", title: \"Connection status\" },\n                                            this.renderStatusLight()\n                                        ),\n                                        \"\\xA0\",\n                                        _react2.default.createElement(\n                                            \"span\",\n                                            { className: \"text\", id: \"uptime\" },\n                                            info.uptime\n                                        )\n                                    )\n                                )\n                            )\n                        )\n                    )\n                ),\n                _react2.default.createElement(\n                    \"div\",\n                    { id: \"no-connection-modal\", className: \"modal\", tabIndex: \"-1\", role: \"dialog\" },\n                    _react2.default.createElement(\n                        \"div\",\n                        { className: \"modal-dialog modal-sm\", role: \"document\" },\n                        _react2.default.createElement(\n                            \"div\",\n                            { className: \"modal-content\" },\n                            _react2.default.createElement(\n                                \"div\",\n                                { className: \"row error-message\" },\n                                _react2.default.createElement(\n                                    \"div\",\n                                    { className: \"col-xs-12\" },\n                                    _react2.default.createElement(\"br\", null),\n                                    _react2.default.createElement(\n                                        \"h4\",\n                                        null,\n                                        \"Unable to connect to server\"\n                                    ),\n                                    _react2.default.createElement(\n                                        \"p\",\n                                        null,\n                                        this.state.errorText ? \"Error: \" + this.state.errorText : null\n                                    )\n                                )\n                            )\n                        )\n                    )\n                )\n            );\n        }\n    }]);\n\n    return PageTitle;\n}(_react2.default.Component);\n\n//# sourceURL=webpack:///./components/PageTitle.jsx?");
```

Comment:
Does Travis check this is regenerated and up to date?

Reply:
Yes it does.